src/main/frontend/handler/repo.cljs

```diff
@@ -417,27 +382,27 @@
                                              (string/includes? (str error) "404"))]
                          (cond
                            (and permission? (not fallback?))
-                           (request-app-tokens!
+                           (helper/request-app-tokens!
```

Comment:
We can remove this part because the token is just requested from the server which should not be expired.